cil_download.rb

```diff
@@ -15,28 +15,44 @@
 results = IO.readlines(ARGV[2])
 
 results.each do |json_record|
-  json_record = json_record.strip
-  identifier = json_record.split('/').last.split('.')[0]
-
-  FileUtils.copy_file(json_record, "#{data_path}/#{identifier}.json")
-  file = File.read(json_record)
-  metadata = JSON.parse(file)
-
-  video_format = metadata['CIL_CCDB']['Data_type']['Video']
-  content_files = metadata['CIL_CCDB']['CIL']['Image_files']
-  if content_files
-    content_files.each do |file|
-      file_path = file['File_path']
-      content_base_url = video_format ? video_base_url : image_base_url
-      `curl -s -o #{content_file_path}/#{file_path} #{content_base_url}/#{identifier}/#{file_path}`
+  begin
+    json_record = json_record.strip
+    identifier = json_record.split('/').last.split('.')[0]
+
+    json_file = "#{data_path}/#{identifier}.json"
+    if !File.exists?(json_file)
```

Comment:
would it be worth considering flipping this to go to the next record immediately if the file does exist?

Reply:
I was worry about in some case that the json file was downloaded while it failed to download some or all of the content files. So I think it should be safer to check it one by one.

Reply:
Ok that makes good sense in the case of the JSON files, since that would prevent the situation you're mentioning. Thanks for clarifying!